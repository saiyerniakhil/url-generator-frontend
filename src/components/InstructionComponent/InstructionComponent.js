const InstructionComponent = () => {
  return (
    <div data-testid="instruction-test" className="text-left border">
      <h4 className="px-3 pt-3">How does it work?</h4>
      <hr style={{ width: "90%" }} />
      <ul>
        <li>
          Enter Link/Message of your choice{" "}
          <span role="img" aria-label="typeing">
            ⌨
          </span>
        </li>
        <li>
          Set the time{" "}
          <span role="img" aria-label="time">
            ⌚
          </span>{" "}
          you want the message to disappear{" "}
          <span role="img" aria-label="charm">
            ✨
          </span>{" "}
          after
        </li>
        <li>
          Submit the form{" "}
          <span role="img" aria-label="mail">
            📪
          </span>
        </li>
        <li>Therafter you'll get the link to the message</li>
        <li>
          Share the link with you friends{" "}
          <span role="img" aria-label="friends">
            👨‍👧‍👦
          </span>
          , after the specified time, you'll not be able to access{" "}
          <span role="img" aria-label="not">
            🚫
          </span>{" "}
          that message
        </li>
      </ul>
    </div>
  );
};

export default InstructionComponent;
