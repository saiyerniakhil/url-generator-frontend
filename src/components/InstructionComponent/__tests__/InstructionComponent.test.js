import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import InstructionComponent from '../InstructionComponent';

test("Instruction component renders correctly", () => {
    expect(<InstructionComponent/>).toMatchSnapshot();
})