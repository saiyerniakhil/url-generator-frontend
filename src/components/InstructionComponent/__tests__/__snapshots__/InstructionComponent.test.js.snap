// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instruction component renders correctly 1`] = `<InstructionComponent />`;
